src/test/javascript/spec/component/participation-submission/participation-submission.component.spec.ts

```diff
@@ -104,6 +106,29 @@ describe('ParticipationSubmissionComponent', () => {
         restore();
     });
 
+    it('Should return empty commit url if participation has no repository url', fakeAsync(() => {
```

Comment:
Why is this async, you are not using tick inside?

Reply:
Good catch, I copy-pasted the `it` incorrectly